source/network/network.go

```diff
@@ -148,7 +148,7 @@ func readIfaceInfo(path string, attrFiles []string) nfdv1alpha1.InstanceFeature
 	for _, attrFile := range attrFiles {
 		data, err := os.ReadFile(filepath.Join(path, attrFile))
 		if err != nil {
-			if !os.IsNotExist(err) {
+			if !os.IsNotExist(err) && !strings.Contains(err.Error(), "invalid argument") {
```

Comment:
Thanks @mantramantra12 for the contribution. I'd suggest to use `errors.Is()` to check the error code directly instead of relying on string comparison

```suggestion
			if !os.IsNotExist(err) && !errors.Is(err, syscall.EINVAL) {
```

/ok-to-test

Reply:
I agree this is much better - didn't know about `syscall.EINVAL`